src/Core/Components/Dialog/ContentComponents/MessageBox.razor

```diff
@@ -8,8 +8,7 @@
             <FluentIcon Value="@Content.Icon" Color="@Content.IconColor" Width="24px" Class="icon" />
         }
         <div>
-            @Content.Message
```

Comment:
The `MessageBoxContent.Message` is obsolete but always present. So I should keep this line.